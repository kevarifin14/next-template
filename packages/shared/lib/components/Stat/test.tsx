import { Stat } from ".";
import { render } from "@testing-library/react";

describe("Stat", () => {
  const renderStat = (props = {}) => render(<Stat {...props} />);

  it("matches snapshot", () => {
    const { baseElement } = renderStat();
    expect(baseElement).toMatchSnapshot();
  });
});
