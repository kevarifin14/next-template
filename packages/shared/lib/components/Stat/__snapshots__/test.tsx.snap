// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Stat matches snapshot 1`] = `
<body>
  <div>
    <div
      class="prose dark:prose-invert"
    >
      <p
        class="m-0 text-sm font-medium truncate"
      />
      <h2
        class="mt-1 text-3xl font-semibold"
      />
    </div>
  </div>
</body>
`;
